docs/RELEASE_NOTES.md

```diff
@@ -27,17 +27,23 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 
 -->
-
 <!--
-### 1.6.0 -  Oct 8, 2019
+### 1.7.0 -  Oct 17, 2019
```

Comment:
is the date correct?

Reply:
yes, its commented out for the future release (as an example)